### PR TITLE
Updated Docker Rules

### PR DESCRIPTION
**Pull request by terenc3t**
Updated docker rules to use manual tag. `bazel build //...` in mac worked fine after 👍

### CODE REVIEW

File: beacon-chain/BUILD.bazel [hunk 1]
```diff
@@ -44,6 +45,7 @@ container_push(
     registry = "gcr.io",
     repository = "prysmaticlabs/prysm/beacon-chain",
     tag = "latest",
```
Comment (rawfalafel) [inline beacon-chain/BUILD.bazel]: Should this be `tags = ["manual","latest"]`?

Comment (terenc3t) [reply beacon-chain/BUILD.bazel]: Right now they are:
```
tag = "latest",
tags = ["manual"],
```
should it just be?
```
tags = ["manual","latest"]
```

Comment (rawfalafel) [reply beacon-chain/BUILD.bazel]: Nevermind, what you have looks correct. [tag](https://github.com/bazelbuild/rules_docker#container_pull-1) is specifc to `rules_docker`, whereas [tags](https://docs.bazel.build/versions/master/be/common-definitions.html#common-attributes) applies to all rules. Would be nice if attributes were namespaced with prefixes.

**Comment by prestonvanloon** [comment]
😍

**Pull request merged**